tests/E2E/test/campaigns/common_scenarios/welcome.js

```diff
@@ -0,0 +1,15 @@
+const {OnBoarding} = require('../../selectors/BO/onboarding.js');
+
+let promise = Promise.resolve();
+
+module.exports = {
+    findAndCloseWelcomeModal: function () {
+        scenario('Close the onboarding modal if exist ', client => {
+            test('should close the onboarding modal if exist', () => {
+              return promise
+                .then(() => client.isVisible(OnBoarding.welcome_modal))
+                .then(() => client.closeBoarding(OnBoarding.popup_close_button))
+            });
+        }, 'order');
+    }
```

Comment:
I'm not an expert of mocha and js tests so I'm not sure, but it seems to me the `'order'` parameter was not always the same in the previous tests (its value could be `installation`) depending on the context
so shouldn't there be a parameter in your `findAndCloseWelcomeModal` to specify this parameter?